src/command.c

```diff
@@ -873,21 +885,21 @@ static uint32_t dv_convert_ipu_v0_fill_cmb(
 	cmd_buf[i++] = fmt_and_flag; // rdFmt, wrFmt, rdEn, faEn, alpha
 	cmd_buf[i++] = 0x0298; // Write to 0x298
 	cmd_buf[i++] = cnv; // conversion to fp16
-	if(cmd->fmt_tex == 7 && cmd->use_tex != 0) {
+	if (cmd->fmt_tex == 7 && cmd->use_tex != 0) {
 		for(j = 0; j < cmd->ncolor_lut; j++) {
 			cmd_buf[i++] = 0x01f0; // Write to 0x01f0
 			cmd_buf[i++] = cmd->lut[j];
 		}
 	}
 	cmd_buf[i++] = 0x0; // end
 	cmd_buf[i++] = 0; // end
-	
+
 	return i;
 }
 
 // If already a command is converted and not executed yet, -EBUSY is returned.
 static int dv_convert_ipu_v0(struct device *dev, struct dmp_cmb *cmb,
-			    struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
+		struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket